prombench/docs/eks.md

```diff
@@ -0,0 +1,113 @@
+# Prombench in EKS
+
+Run prombench tests in [Elastic Kubernetes Service](https://aws.amazon.com/eks/).
+
+## Setup prombench
+
+1. [Create the main node](#create-the-main-node)
+2. [Deploy monitoring components](#deploy-monitoring-components)
+
+### Create the Main Node
+
+---
+
+- Create [security credentials](https://docs.aws.amazon.com/general/latest/gr/aws-sec-cred-types.html) on AWS. Put the credentials into a `yaml` file as follows:
+```yaml
+accesskeyid: <Amazon access key>
+secretaccesskey: <Amazon access secret>
+```
+- Create a [VPC](https://docs.aws.amazon.com/eks/latest/userguide/create-public-private-vpc.html) with public subnets.
```

Comment:
Should we mention that the name should be same as our cluster name?

Reply:
No, But it should contain the tag mentioned.